doc-content/enterprise-only/openshift/environment-managed-deploy-proc.adoc

```diff
@@ -30,12 +30,15 @@ In this command line:
 . Set the following parameters as necessary:
 ** *{CENTRAL} Server Keystore Secret Name* (`{CENTRAL_CAPITAL_UNDER}_HTTPS_SECRET`): The name of the secret for {CENTRAL}, as created in <<secrets-central-create-proc>>.
 ** *KIE Server Keystore Secret Name* (`KIE_SERVER_HTTPS_SECRET`): The name of the secret for {KIE_SERVER}, as created in <<secrets-kie-create-proc>>.
+** *Smart Router Keystore Secret Name* (`KIE_SERVER_ROUTER_HTTPS_SECRET`): The name of the secret for Smart Router, as created in <<secrets-kie-create-proc>>.
```

Comment:
I think this section reference should be "secrets-smartrouter-create-proc" to link to section 2.4 (instead of 2.2)